### PR TITLE
Ci 404 follow up changes because worksheet changed

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/Esfa.Ofsted.Inspection.Client/Services/OverallEffectivenessProcessor.cs [hunk 1]
```diff
@@ -20,7 +20,8 @@ internal class OverallEffectivenessProcessor: IOverallEffectivenessProcessor
                 case "9":
                     return OverallEffectiveness.RemainedGoodAtAShortInspectionThatDidNotConvert;
                 case "-":
-                case null:
+                case "NULL":
+                case "null":
                     return OverallEffectiveness.NotJudged;
```
Comment (arvinthseran) [inline src/Esfa.Ofsted.Inspection.Client/Services/OverallEffectivenessProcessor.cs]: can we add `case "Null":` as well?

File: src/Esfa.Ofsted.Inspection.UnitTests/OverallEffectivenessProcessorTests.cs [hunk 2]
```diff
@@ -13,8 +13,8 @@ public class OverallEffectivenessProcessorTests
         [TestCase("4", OverallEffectiveness.Inadequate)]
         [TestCase("9", OverallEffectiveness.RemainedGoodAtAShortInspectionThatDidNotConvert)]
         [TestCase("-", OverallEffectiveness.NotJudged)]
-        [TestCase(null, OverallEffectiveness.NotJudged)]
-       
+        [TestCase("NULL", OverallEffectiveness.NotJudged)]
+        [TestCase("null", OverallEffectiveness.NotJudged)]
         public void ShouldReturnStringModifiedForUrlUsage(string inputText, OverallEffectiveness expectedEffectiveness)
```
Comment (arvinthseran) [inline src/Esfa.Ofsted.Inspection.UnitTests/OverallEffectivenessProcessorTests.cs]: `[TestCase("Null", OverallEffectiveness.NotJudged)]` ?

**Review by arvinthseran — APPROVED** [review]

File: src/Esfa.Ofsted.Inspection.UnitTests/OverallEffectivenessProcessorTests.cs (hunk 2 above)
Comment (MarkFCain) [reply src/Esfa.Ofsted.Inspection.UnitTests/OverallEffectivenessProcessorTests.cs]: Yeah, I'll do this after I do the logs

File: src/Esfa.Ofsted.Inspection.Client/Services/GetOfstedDetailsFromExcelPackageService.cs [hunk 3]
```diff
@@ -145,18 +147,28 @@ private static int FindStartingLineNumber(ExcelWorksheet keyWorksheet)
...
+        return datePublished;
         }
 
         private OverallEffectiveness? ProcessOverallEffectivenessForError(string value, InspectionError error)
         {
             var overallEffectivenessString = value;
             var overallEffectiveness = _overallEffectivenessProcessor.GetOverallEffectiveness(overallEffectivenessString);
             error.OverallEffectiveness = overallEffectivenessString;
+            if (overallEffectiveness==null)
+                error.Message = error.Message + $@"Invalid value for Overall Effectiveness [{ error.OverallEffectiveness}]; ";
```
Comment (scottcowan) [inline src/Esfa.Ofsted.Inspection.Client/Services/GetOfstedDetailsFromExcelPackageService.cs]: could you add some { } here

**Review by scottcowan — CHANGES_REQUESTED** [review]
feel free to merge after this

**Pull request merged**